test/test_integration.py

```diff
@@ -434,6 +442,31 @@ def test_vault_github_login_from_env():
                          network=True)
 
 
+def test_error_if_orderly_not_initialised():
+    path = "config/basic"
+    options = {"orderly": {"initial": None}}
+    cfg = build_config(path, options=options)
+    # ensure this test behaves sensibly if state is a bit messy:
+    with docker_client() as cl:
+        remove_volume(cl, cfg.volumes["orderly"])
+    try:
+        with pytest.raises(Exception,
+                           match="Orderly volume not initialised"):
+            res = orderly_web.start(path, options=options)
+        # now we have this container up we can initialise it anyway:
+        container = cfg.get_container("orderly")
```

Comment:
Would this second part make more sense as a separate test, test_can_initialise_from_volume?

Reply:
yes, agreed - e0bc886